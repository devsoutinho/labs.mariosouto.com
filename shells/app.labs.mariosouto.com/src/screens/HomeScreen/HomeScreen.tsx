import { Box, Text } from "@devsoutinho/sknui/native";

export function HomeScreen() {
return (
<Box styleSheet={{
flex: 1,
backgroundColor: '#000',
alignItems: 'center',
justifyContent: 'center',
}}>
<Text styleSheet={{
color: '#fff',
}}>
Ol√° pessoas!
</Text>
</Box>
);
}