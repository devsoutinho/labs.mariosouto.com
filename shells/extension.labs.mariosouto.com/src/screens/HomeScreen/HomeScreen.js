import React from "react";
import { Box, Text } from "@devsoutinho/sknui";
export function HomeScreen() {
  return React.createElement(
    Box,
    {
      styleSheet: {
        flex: 1,
        backgroundColor: "#000",
        alignItems: "center",
        justifyContent: "center",
      },
    },
    React.createElement(
      Text,
      {
        styleSheet: {
          color: "#fff",
        },
      },
      "Ol\u00E1 pessoas!"
    )
  );
}
