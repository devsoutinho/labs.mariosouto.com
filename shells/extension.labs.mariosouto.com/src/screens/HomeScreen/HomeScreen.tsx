import React from "react";
import { Box, Text } from "@devsoutinho/sknui/web";

export function HomeScreen() {
  const [message, setMessage] = React.useState("");
  React.useEffect(() => {
    chrome.storage.local.get(["wordCountApp"], (result) => {
      const { length, totalWords } = result.wordCountApp || {};
      setMessage(
        `Selected text length: ${length} | Selected text total words: ${totalWords}`
      );
    });
  }, []);

  return (
    <Box
      styleSheet={{
        width: "320px",
        flex: 1,
        backgroundColor: "#000",
        alignItems: "center",
        justifyContent: "center",
      }}
    >
      <Text
        styleSheet={{
          color: "#fff",
        }}
      >
        {message}
      </Text>
    </Box>
  );
}
