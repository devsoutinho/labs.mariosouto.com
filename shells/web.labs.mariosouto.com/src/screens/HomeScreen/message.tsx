export const message = "OlÃ¡ pessoas!";
