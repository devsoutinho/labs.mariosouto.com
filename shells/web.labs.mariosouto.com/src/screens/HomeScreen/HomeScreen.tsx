import React from "react";
import { Box, Text, Form, InputText, Button, Icon } from "@devsoutinho/sknui/web";
import { message } from "@src/screens/HomeScreen/message";

export function HomeScreen() {
  return (
    <Box
      styleSheet={{
        flex: 1,
        alignItems: "center",
        justifyContent: "center",
      }}
    >
      <Form onSubmit={() => {
        window.location.href = "/dashboard";
      }}>
        <InputText label="Email"  />

        <Button
          type="submit"
          suffix={<Icon name="arrow_left" />}
        >
          Entrar
        </Button>
        <Box
          styleSheet={{
            marginTop: "16px",
          }}
        >
          <Text>
            Ou fa√ßa login com
          </Text>
          <Box>
            <Button>
              Github
            </Button>
          </Box>
        </Box>
      </Form>
    </Box>
  );
}
