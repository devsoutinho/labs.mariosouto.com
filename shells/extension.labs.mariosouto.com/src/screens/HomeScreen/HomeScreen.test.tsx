import { HomeScreen } from "./HomeScreen";

describe("<HomeScreen />", () => {
  it("true is true", () => {
    // eslint-disable-next-line no-console
    console.log(HomeScreen);
    expect(true).toBe(true);
  });
});
