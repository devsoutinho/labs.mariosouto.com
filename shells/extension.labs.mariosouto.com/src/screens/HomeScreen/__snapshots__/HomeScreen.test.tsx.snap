// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders as expected 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  overflow: hidden;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  background-color: #000;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
}

<div>
  <div
    class="emotion-0"
  >
    <p
      style="color: rgb(255, 255, 255);"
    >
      Ol√° pessoas!
    </p>
  </div>
</div>
`;
